bin/.gitignore

```diff
@@ -1,26 +1 @@
-/input/
```

Comment:
Let's revert this so we don't commit a bunch of images from /bin

Reply:
Reverted